lib/streamlit/runtime/state/session_state.py

```diff
@@ -534,7 +534,6 @@ def on_script_finished(self, widget_ids_this_run: set[str]) -> None:
             run. Any widget state whose ID does *not* appear in this set
             is considered "stale" and will be removed.
         """
-        self._reset_triggers()
```

Comment:
Why can we delete this, will the triggers be still cleared up correctly?

Reply:
It's safe because we also call it in the script startup (`on_script_will_run` or something like that), so this was redundant.